training/watch.md

```diff
@@ -0,0 +1,29 @@
+---
+---
+
+# 3. When do you watch spam?
+Adding websites and keywords to be watched by SmokeDetector is a big part of being a privileged
```

Comment:
I'd phrase this as "Adding websites and keywords to SmokeDetector's watchlists is...".

Reply:
Completed.